gateway/platforms/vertx3/vertx3/src/main/java/io/apiman/gateway/platforms/vertx3/connector/ConnectorFactory.java

```diff
@@ -143,4 +147,8 @@ private void setAttributesFromApiEndpointProperties(Api api, ApimanHttpConnector
     public IConnectorConfig createConnectorConfig(ApiRequest request, Api api) {
         return new VertxConnectorConfig();
     }
+
+    private void applyProxyOptions(ApimanHttpConnectorOptions opts, HttpClientOptions vxClientOptions) {
```

Comment:
This is only applying system property proxy options for now, right? I think this name could be misleading otherwise, and I will change it unless you think otherwise.